app/src/ui/diff/side-by-side-diff.tsx

```diff
@@ -916,8 +925,9 @@ function getDiffRowsFromHunk(
     if (line.type === DiffLineType.Hunk) {
       rows.push({
         type: DiffRowType.Hunk,
-        content: line.content,
+        content: line.text,
```

Comment:
This is what fixes the issue with the first `@` missing. `line.content` refers to the content of the line in a diff (which means the initial `+` or `-` is removed).